source/_integrations/remootio.markdown

```diff
@@ -24,3 +24,29 @@ To enable the API access on your Remootio device you need to access it using the
 in the _Device software_ section of the device settings and enable the API with logging.
 
 {% include integrations/config_flow.md %}
+
+## Events
+
+The integration fires the event `remootio_left_open` if the Remootio controlled garage door or garden gate has been left open for 5 minutes.
```

Comment:
Can we use device actions instead?

Reply:
According to the [documentation](https://developers.home-assistant.io/docs/device_automation_action/) device actions allow a user to have a device do something. 

In this case the device fires an event.

Reply:
> In this case the Remootio device fires an event if the controlled garage door or garden gate has been left open for 5 minutes. 

Device actions are using events under the hood, but provide UI controls to interact with those.

<https://developers.home-assistant.io/docs/device_automation_index>

Reply:
If you mean that this is really needed for the first version of the integration, please make then an request for change in home-assistant/core#63155. Thank you!

Reply:
> Maybe to much for a first version indeed. however, lets at least not use json here.

Sorry, I don't understand. What you mean with _not use json here_?

Reply:
Nevermind it is indeed the raw event output.  👍